lti_store/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class LTIVersion(models.TextChoices):
+    LTI_1P1 = "lti_1p1", _("LTI 1.1")
+    LTI_1P3 = "lti_1p3", _("LTI 1.3")
```

Comment:
@tecoholic Remove the LTI 1.3 version choice from here for now.

Reply:
Done :)